src/app/pages/website/contact/contact.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from "@angular/forms";
+
+@Component({
+  selector: 'littil-contact',
+  templateUrl: './contact.component.html',
+  styleUrls: ['./contact.component.scss']
+})
+export class ContactComponent implements OnInit {
+
+  contactForm?: FormGroup;
+
+  constructor(
+    private fb: FormBuilder,
+  ) {}
+
+  ngOnInit(): void {
+   this.createContactForm();
+  }
+
+  createContactForm(): void {
+    this.contactForm = this.contactForm = this.fb.group({
```

Comment:
Je kunt ook gewoon een FromGroup({}) maken zonder de formBuilder? Het formulier is niet complex genoeg en je voorkomt zo dat je "this.contactForm?" moet doen overal.

Reply:
Voorbeeld:
`registerForm: FormGroup = new FormGroup({
    email: new FormControl('', [Validators.required, Validators.email]),
  });`